UI/addParty.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8">
+    	<meta name="viewport" content="width=device-width,initial-scale=1.0">
+		<title>politico : Create party</title>
+		<link href="https://fonts.googleapis.com/css?family=Lato:100,300,400,700,900" rel="stylesheet">
+		<link href="assets/css/styles.css" type="text/css" rel="stylesheet" media="all" />
+		<link href="assets/css/createparty.css" type="text/css" rel="stylesheet" media="all" />
+		<link rel="shortcut icon" type="image/x-icon" href="./assets/images/logo.gif">
+	</head>
+	<body>
+			<nav class="politico-nav">
+					<div class="nav-container">
+						<div class="nav-left">
+							<a href="./frontPage.html"><h2><img class="laravel" src="./assets/images/logo.gif" height="40px" width="40px" alt="">  POLITICO</h2></a>
+						</div>
+		
+						<div class="nav-right">
+											
+						<ul class="header">
+								<li class="login-btn">
+									<a href="./signin.html">login</a> 
+								</li>
+								<li class="signup-btn"> 
+								   <a href="./signup.html">sign up</a> 
+								</li>
+								<li>
+										<a href="./signin.html">
+											<button class="nav-button">Logout</button>
+										</a>
+								</li>
+							</ul>
+		
+							
+						</div>
+					</div>
+		
+				</nav>
+		<div class="create-party-area">
+			<div class="container">
+				<div class="create-party-form-area">
+					<h3 class="create-party-form-text">Create Party</h3>
+					<form>
+						<div class="create-party-input-section">
+							<input 
+								type="text" 
+								placeholder="Name of Party"
+								required 
+							/>
+						</div>
+						<div class="create-party-input-section">
+							<textarea required placeholder="Description"l></textarea> 
+						</div>
+						<div class="create-party-form-submit">
+							<button type="submit">Submit</button>
+						</div>
+					</form>
+				</div>
+			</div>
+		</div>
+		<footer class="politico-footer">
+			<p>politico. &copy; 2018</p>
+		</footer>
+	</body>
+</html>
```

Comment:
A new line here, please. Also, kindly format your html, so it looks well. Thanks